src/library/Uno.Toolkit.Material/Styles/Controls/v2/Card.xaml

```diff
@@ -3,24 +3,48 @@
                     xmlns:utu="using:Uno.Toolkit.UI"
                     xmlns:um="using:Uno.Material"
                     xmlns:toolkit="using:Uno.UI.Toolkit">
-
-    <!-- Card Brushes Resources -->
-    <!-- Elevated -->
-    <StaticResource x:Key="MaterialElevatedCardBackground"
-	                ResourceKey="SurfaceBrush" />
-
-    <!-- Filled -->
-    <StaticResource x:Key="MaterialFilledCardBackground"
-	                ResourceKey="SurfaceVariantBrush" />
-    
-    <!-- Outlined -->
-    <StaticResource x:Key="MaterialOutlinedCardBackground"
-	                ResourceKey="SurfaceBrush" />
-    <StaticResource x:Key="MaterialOutlinedCardBorderBrush"
-	                ResourceKey="OutlineBrush" />
-
-    <!--  Card Other Resources  -->
-    <!-- Double -->
+	<ResourceDictionary.ThemeDictionaries>
+		<!-- Card Brushes Theme Resources -->
+		<ResourceDictionary x:Key="Default">
+			<!-- Filled -->
+			<StaticResource x:Key="MaterialFilledCardBackground"
+							ResourceKey="SurfaceBrush" />
+
+			<!-- Outlined -->
+			<StaticResource x:Key="MaterialOutlinedCardBackground"
+							ResourceKey="SurfaceBrush" />
+
+			<!-- Elevated -->
+			<StaticResource x:Key="MaterialElevatedCardBackground"
+							ResourceKey="SurfaceBrush" />
+
+			<!--  Card Other Resources  -->
+			<StaticResource x:Key="MaterialOutlinedCardBorderBrush"
+							ResourceKey="OutlineBrush" />
+		</ResourceDictionary>
+
+		<ResourceDictionary x:Key="Light">
+			<!-- Filled -->
+			<StaticResource x:Key="MaterialFilledCardBackground"
+							ResourceKey="SurfaceBrush" />
+
+			<!-- Outlined -->
+			<StaticResource x:Key="MaterialOutlinedCardBackground"
+							ResourceKey="SurfaceBrush" />
+
+			<!-- Elevated -->
+			<StaticResource x:Key="MaterialElevatedCardBackground"
+							ResourceKey="SurfaceBrush" />
+
+			<!--  Card Other Resources  -->
+			<StaticResource x:Key="MaterialOutlinedCardBorderBrush"
+							ResourceKey="OutlineBrush" />
+		</ResourceDictionary>
+
+		<ResourceDictionary x:Key="HighContrast" />
```

Comment:
Should remove this, otherwise the resource resolution won't fallback to the Default dictionary when searching for keys that aren't defined while in HighContract mode

Reply:
Removed